nixos/modules/services/web-apps/writefreely.nix

```diff
@@ -0,0 +1,102 @@
+{config, lib, pkgs, ...}:
+
+with lib;
+
+let
+  cfg = config.services.writefreely;
+  configFile = pkgs.writeText "config.ini" (generators.toINI {} cfg.settings);
+  settingsFormat = pkgs.formats.json {};
+in {
+  options.services.writefreely = {
+    enable = mkEnableOption "WriteFreely, a clean, Markdown-based publishing platform made for writers. Write together, and build a community.";
+
+    package = mkOption {
+      type = types.package;
+      default = pkgs.writefreely;
+      description = ''
```

Comment:
could you mark all descriptions with `mdDoc`? that will make it easier to fully transition to markdown docs :)